specification.rst

```diff
@@ -47,6 +47,25 @@ throughout this specification when referring to CSL elements, but is generally
 omitted in favor of a default namespace declaration (set with
 the ``xmlns`` attribute) on the root ``cs:style`` or ``cs:locale`` element.
 
+Whitespace Handling
+-------------------
+
+CSL styles are valid XML, but CSL processors MUST NOT normalize attribute values 
+by trimming leading or trailing whitespace from attributes which define text that 
+is intended for output:
+
+- after-collapse-delimiter
+- cite-group-delimiter
+- delimiter
+- initialize-with
+- name-delimiter
+- names-delimiter
+- prefix
+- range-delimiter
+- sort-separator
+- suffix
+- year-suffix-delimiter
```

Comment:
`- value`

Reply:
I meant like this; I thought you were the one who told me about this feature.

```suggestion
- year-suffix-delimiter
-value
```